docs/pages/reference/user-types.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: User Types
+description: Describes the different types of Teleport users and their properties.
+keywords: [user,idp,sso]
+tocDepth: 3
+---
+
+This guide explains the different kinds of users in Teleport, how they are
+created, and their properties.
+
+## Local Users
+
+Local users are created directly in Teleport. They are not coming from an
+external system like an identity provider.
+
+Local users can be created via the CLI (`tctl users add`), by applying a user
+resource manifest (`tctl create -f user.yaml`) or via the web UI. Their roles
+and traits can be modified directly in Teleport.
+
+Those users can connect to Teleport directly via username/password and/or
+additional authentication factors such as webauthn physical tokens, passkeys or
+One Time Passwords.
+
+Local user login can be disabled via `cluster_auth_preference` or `teleport.yaml`.
+Disabling local authentication is required for [FIPS/FedRAMP compliance
+](../access-controls/compliance-frameworks/fedramp.mdx).
+
+### Special case: Bots
+
+Machine ID provides machines with an identity that can authenticate to the
+Teleport cluster. This identity is known as a bot. Bots are represented in
+Teleport by a user and a role resource and can be created via the
+`tctl bots add` command.
+
+Bots are not logging in like human users with a password, MFA or SSO. They join
+the cluster as Teleport services using [a join method](./join-methods.mdx). They
+can still join even if local auth is disabled. 
+
+<Admonition type="note" title="See also">
+The [Machine ID introduction](../machine-id/introduction.mdx).
+</Admonition>
+
+## SSO users
+
+Those users are imported in Teleport from an external identity provider.
+Teleport contains a user resource representing those users, but it's only a
+representation of a user of a remote system.
+
+Remote users don't log in directly in Teleport, they are redirected to their
+identity provider to perform the login challenge.
+
+Those users cannot be edited via `tctl`, or via any other IaC tooling like
+Terraform or the Teleport Kubernetes Operator. They are managed by 
+
+### Temporary users
+
+Users logging in via GitHub, SAML, or OIDC connectors are created in Teleport
+after a successful login. At that time, their roles and traits are computed
+according to the connector mappings.
+
+Those users are short-lived, they are created when a Teleport session is opened
+and automatically expires after a few days. Those users cannot be edited via
```

Comment:
Also, we should be more precise than "a few days"—what's the TTL?

Reply:
Expiry depends on a lot of factors. The IdP response validity, the max session length of your roles. There is a static hardcoded limit of 30 hours.